includes/NormalizerFactory.php

```diff
@@ -0,0 +1,52 @@
+<?php
```

Comment:
|`line 10`|
|---|
I think this should be either named ObjectNormalizerFactory, or it should return a NormalizerInterface